doc/Quick_Start/README.md

```diff
@@ -20,7 +20,7 @@ RAK5005-O is the WisBlock Base board which can be connected with WisBlock Core a
 
 RAK4631 is the WisBlock Core board which consists of nRF52840 and SX1262. It supports LoRa® and BLE functions, and supply a rich resource MCU so that you can program it if you want.
 
-WisBlock is not only functional testability product in product development verification stage, but also industrial products oriented to mass production. It uses a high-speed connector to ensure the integrity of the signal. At the same time, it is equipped with fastening screw, which can be used in vibration environment. And WisBlock can be used reliably in various civil and industrial scenarios through rigorous reliability tests.
+WisBlock is not only functional test-ability product in product development verification stage, but also industrial products oriented to mass production. It uses a high-speed connector to ensure the integrity of the signal. At the same time, it is equipped with fastening screw, which can be used in vibration environment. And WisBlock can be used reliably in various civil and industrial scenarios through rigorous reliability tests.
```

Comment:
I think this would suit better:
"Wisblock is not only a functional test capable product during the development and verification stage, but also..."